content/posts/about_inlining.md

```diff
@@ -52,36 +53,36 @@ The last one is pro only in certain specific cases, but still pro nonetheless.
 ## Cons
 
 1. It increases the executable size due to code expansion.
-2. C++ inlining is resolved at compile time. This means if you change something in the inlined function, you would need to recompile all source code using it to make sure it (nên để là "the whole project/repo/sourcebase/sourcecode/etc", quá nhiều "it" sẽ gây khó hiểu là nó đang trỏ vào đâu) will be updated.
-3. When used in a header, it makes your header file larger with information which users don’t care.
-4. As mentioned above it increases the executable size, which may cause thrashing in memory. More number of page fault bringing down your program performance.
-5. Sometimes inlining is not useful. For example, in embedded system where large executable size is not preferred at all due to memory constraints.
+2. C++ inlining is resolved at compile time. This means if you change something in the inlined function, you would need to recompile the whole source codes which are using the function to make sure it will be updated.
```

Comment:
[Refactor] Over complicated in "which are using", reduce to "using" is enough to tell what you mean. And in grammar, that's the right one.
[Suggestion] If you still wanna use "which", you should write as "which use". As your sentence is quite long, if you follow this, you need some colon in between them.